### PR TITLE
Add multiprecision for adadelta op

### PR DESCRIPTION
**Pull request by AnnaTrainingG**
### PR types

 New features
### PR changes

 APIs
### Describe

Add multiprecision for adadelta op

### CODE REVIEW

File: paddle/phi/ops/compat/adadelta_sig.cc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#include "paddle/phi/core/compat/op_utils.h"
+
+namespace phi {
+
+KernelSignature AdadeltaOpArgumentMapping(const ArgumentMappingContext& ctx) {
+  if (ctx.IsDenseTensorInput("Grad")) {
+    return KernelSignature(
+        "adadelta",
+        {"Param", "Grad", "AvgSquaredGrad", "AvgSquaredUpdate", "MasterParam"},
+        {"epsilon", "multi_precision"},
```
Comment (zkh2016) [inline paddle/phi/ops/compat/adadelta_sig.cc]: 这里少了rho参数

Comment (AnnaTrainingG) [reply paddle/phi/ops/compat/adadelta_sig.cc]: 已经添加

**Review by jiweibo — DISMISSED** [review]
LGTM for `Dispensable of attr and inout`

**Review by Shixiaowei02 — DISMISSED** [review]

**Review by Shixiaowei02 — DISMISSED** [review]

**Review by jiweibo — DISMISSED** [review]
LGTM for `dispensable inout` and `default attr`

**Review by Shixiaowei02 — APPROVED** [review]

**Review by jiweibo — APPROVED** [review]

**Review by zkh2016 — APPROVED** [review]

**Pull request merged**